src/lib/UserSetting/Setting/FullDateTimeFormat.php

```diff
@@ -81,7 +81,7 @@ public function mapFieldForm(
     protected function getTranslatedName(): string
     {
         return $this->translator->trans(
-            /** @Desc("Long Date & Time format") */
+            /** @Desc("Long date and time format") */
```

Comment:
```suggestion
            /** @Desc("Long Date and Time format") */
```
Date and Time are always capitalized.

Reply:
Discussed face-to-face.